hg-fast-export.sh

```diff
@@ -203,7 +186,7 @@ $(
   } | \
   {
     _e2=0
-    git fast-import $GFI_OPTS --export-marks="$GIT_DIR/$PFX-$SFX_MARKS.tmp" 3>&- || _e2=$?
+    git -c core.ignoreCase=false fast-import $GFI_OPTS --export-marks="$GIT_DIR/$PFX-$SFX_MARKS.tmp" 3>&- || _e2=$?
```

Comment:
We have been supporting mangling to a case-insensitive FS since the beginning (although with sometimes surprising results), so I'm not happy about suddenly removing that functionality. I think that we should force `core.ignoreCase=false` in the tests, but not force that on the user.